tests/integration/helpers/cluster.py

```diff
@@ -1139,6 +1143,26 @@ def setup_postgres_cmd(self, instance, env_variables, docker_compose_yml_dir):
         ]
         return self.base_postgres_cmd
 
+    def setup_postgres11_cmd(self, instance, env_variables, docker_compose_yml_dir):
+        self.base_cmd.extend(
+            ["--file", p.join(docker_compose_yml_dir, "docker_compose_postgres_11.yml")]
+        )
+        env_variables["POSTGRES_PORT"] = str(self.postgres_port)
+        env_variables["POSTGRES_DIR"] = self.postgres_logs_dir
+        env_variables["POSTGRES_LOGS_FS"] = "bind"
+
+        self.with_postgres11 = True
+        self.base_postgres_cmd = [
```

Comment:
if `with_postgres && with_postgres11` you overwrite `self.base_postgres_cmd` twice, they should be different. Now it seems in this case you always start either new postgres or postgres11 but not both, but `wait_postgres_to_start` and `wait_postgres11_to_start` are both called anyway in this case while you have started only one of them.

Reply:
Ah good catch. Thanks a lot.